UpsertUserDataProcessing/handler.ts

```diff
@@ -81,54 +84,63 @@ export function UpsertUserDataProcessingHandler(
     }
     const maybeRetrievedUserDataProcessing =
       errorOrMaybeRetrievedUserDataProcessing.value;
-    const computedStatus = maybeRetrievedUserDataProcessing.fold(
-      UserDataProcessingStatusEnum.PENDING,
-      retrieved => {
-        return retrieved.status === UserDataProcessingStatusEnum.WIP
-          ? retrieved.status
-          : UserDataProcessingStatusEnum.PENDING;
-      }
-    );
-    const userDataProcessing: UserDataProcessing = {
-      choice: upsertUserDataProcessingPayload.choice,
-      createdAt: new Date(),
-      fiscalCode,
-      status: computedStatus,
-      userDataProcessingId: id
-    };
-    const errorOrUpsertedUserDataProcessing = await userDataProcessingModel.createOrUpdateByNewOne(
-      userDataProcessing
-    );
-    if (isLeft(errorOrUpsertedUserDataProcessing)) {
-      const { body } = errorOrUpsertedUserDataProcessing.value;
+    const errorOrComputedStatus = maybeRetrievedUserDataProcessing.fold<
+      Either<IResponseErrorConflict, UserDataProcessingStatusEnum>
+    >(right(UserDataProcessingStatusEnum.PENDING), retrieved => {
+      return retrieved.status === UserDataProcessingStatusEnum.CLOSED
+        ? right(UserDataProcessingStatusEnum.PENDING)
+        : left(
+            ResponseErrorConflict(
+              "An other request is already PENDING or WIP for this User"
+            )
+          );
+    });
 
-      context.log.error(`${logPrefix}|ERROR=${body}`);
+    return errorOrComputedStatus.bimap(
+      conflicError => conflicError,
```

Comment:
```suggestion
      identity,
```

(there's a typo anyway conclic -> conflict)

since the left part is unchanged, you can just `return errorOrComputedStatus.map()`